docs/UserGuide.md

```diff
@@ -72,14 +72,22 @@ return back to [Table of Contents](#table-of-contents)
 
 ## Installing MediConnect
 
-1. Ensure you have Java `11` or above installed in your Computer.
+1. Ensure you have Java `11` or above installed in your Computer. If you do not have it installed yet, refer to the following guides based on your operating system:
+
+    [Windows](https://docs.oracle.com/en/java/javase/11/install/installation-jdk-microsoft-windows-platforms.html#GUID-371F38CC-248F-49EC-BB9C-C37FC89E52A0)
+
+    [macOS](https://docs.oracle.com/en/java/javase/11/install/installation-jdk-macos.html#GUID-F575EB4A-70D3-4AB4-A20E-DBE95171AB5F)
+
+    [Linux](https://docs.oracle.com/en/java/javase/11/install/installation-jdk-linux-platforms.html#GUID-4A6BD592-1840-4BB4-A758-4CD49E9EE88B)
 
 1. Download the latest `mediconnect.jar` from [here](https://github.com/AY2324S1-CS2103T-T08-1/tp/releases).
 
 1. Copy the file to the folder you want to use as the _home folder_ for your MediConnect.
 
 1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar mediconnect.jar` command to run the application.<br>
```

Comment:
You wanna break this into 2 steps? Then for the terminalCommand image you just annotate the steps accordingly

Reply:
yes boss